### ISSUE TITLE
Error executing htmlproof

### ISSUE THREAD

**Issue by duanefields** [gjtorikian/html-proofer#161]
On ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-darwin14]

````
htmlproof 2.0.1 | Error:  `@data-toggle' is not allowed as an instance variable name

**Comment by duanefields** [comment]
This seems to happen on html files that have data-toggle attributes, and it crashes the program, and exits  immediately

**Comment by duanefields** [comment]
This should be resolved by #158, but I don't think it's in the gem yet

**Comment by gjtorikian** [comment]
It is out now. Try version `2.0.2`.
